README.rst

```diff
@@ -0,0 +1,311 @@
+===============================================
+Object Oriented MicroMagnetic Framework (OOMMF)
+===============================================
+
+
++-----------------------------------+-----------------------------------+
+| Description                       | Badge                             |
++===================================+===================================+
+| Build (this repo) on latest Ubuntu| |on-ubuntu-latest|                |
++-----------------------------------+-----------------------------------+
+| Build (this repo) on OSX          | |badge-osx|                       |
++-----------------------------------+-----------------------------------+
+| Release on conda-forge            | |Anaconda-Server Badge|           |
++-----------------------------------+-----------------------------------+
```

Comment:
Add a link to the Docker image on Docker hub?

Reply:
I have added a link to the workflow (via a badge). The docker hub link is given in the section on Docker  further below in the document.

Reply:
done.